src/common/sonarqube-v5/sonarqube/__tests__/Endpoint-test.ts

```diff
@@ -35,7 +35,7 @@ it("On SonarCloud password is always null", () => {
   expect(result.auth.password).toEqual("");
 });
 
-// VSTS-250
+// SONARAZDO-250
```

Comment:
lots of ticket references :))

Reply:
it did cross my mind to drop these refs tbh 😇